04 Strategy Library/207 Value Effect within Countries/01 Introduction.html

```diff
@@ -0,0 +1,5 @@
+<p>
+  The equity valuation has a predictive ability for future equity returns. There are various methodologies to evaluate whether
+  the equity is undervalued or overvalued using metrics like price-to-earnings (P/E), return on equity (ROE), dividend yield, book-to-equity and so on.
+  In this algorithm, we are going to use a ten-year normalized earnings metrics invented by Yale University professor Robert Shiller to find the fair value of the equity market.
```

Comment:
"The equity valuation has a predictive ability for future equity returns." -> Equity valuation may be a predictive signal for future equity return." 

present tense -> we use a ten year...